sr_multi_moveit/sr_multi_moveit_config/scripts/generate_robot_srdf.py

```diff
@@ -492,32 +480,53 @@ def create_move_group_state(self, group_name, group_state_name):
             combined_move_group_state["joint_angles"]
         return True
 
-    def add_bimanual_arm_groups(self, group_1, group_2, hands=True):
+    def add_bimanual_arm_groups(self, manipulators):
+        self.add_comments(comments=["Bimanual arm groups without hands"])
+        # Add two arms (no hands) group
+        self.add_move_group_combining_others(
+            'two_arms', [manipulators[0].arm.internal_name, manipulators[1].arm.internal_name])
+        if manipulators[0].has_hand or manipulators[0].has_hand:
```

Comment:
Shouldn't this be:
```suggestion
        if manipulators[0].has_hand or manipulators[1].has_hand:
```

Reply:
Oops, fixed in https://github.com/shadow-robot/sr_interface/pull/807/commits/7ae6013bee87b2362c5ace6f6cefbac4782a4a89